backend/python/app/services/implementations/entity_service_mg.py

```diff
@@ -45,7 +50,10 @@ def update_entity(self, id, entity):
 
     def delete_entity(self, id):
         try:
+            file_name = Entity.objects.get(id=id).file_name
```

Comment:
Mongo should return the entity when deleted, so you shouldn't need to make an extra query.